spec/spec_helper.rb

```diff
@@ -90,9 +89,15 @@
   config.fail_fast = ENV['FAIL_FAST'] || false
   config.order = "random"
 
-  Capybara.javascript_driver = :poltergeist
-  Capybara.default_max_wait_time = 5
-  Capybara.register_driver :poltergeist do |app|
-    Capybara::Poltergeist::Driver.new(app, timeout: 1.minute, phantomjs_options: ['--load-images=no'])
+  require 'selenium/webdriver'
```

Comment:
what about using https://github.com/solidusio/solidus_support/blob/master/lib/solidus_support/extension/feature_helper.rb ?

Reply:
It's now available in solidus_support 0.2.2 (latest solidus_support version).